### ISSUE TITLE
Can't understand, how we can deposit 10 ether collateral, and mint 100 ether DSC in test contract?

### ISSUE THREAD

**Issue by Blngogo** [Cyfrin/foundry-defi-stablecoin-f23#45]
in DSCEngineTest contract has these variables: 
uint256 amountCollateral = 10 ether;
uint256 amountToMint = 100 ether;

function testCanMintDsc() public depositedCollateral {
        vm.prank(user);
        dsce.mintDsc(amountToMint);

        uint256 userBalance = dsc.balanceOf(user);
        assertEq(userBalance, amountToMint);
    }
in "depositedCollateral" modifier, "amountCollateral" is deposited, so how we can mint amount 10 times higher than the collateral, how this is not breaking the health factor and passes? Can someone explain, please?

**Comment by omarhibshi** [comment]
I looked into this and I have produced the following  log from the test to show that every thing looks to be in order. 
I think the confusion stems from the meaning of (10 ether) and (100 ether). 
It's  just a number format. 10 ether of WETH (collateral) is worth way more tan 100 ether of DSC (debt).

Pre - MintDSC  
  ----------------------------------------------------
  USER DSC balance                    0
  USER weth balance                   0
  USER's TotalDscMinted               0
  USER's TotalCollateralValueInUsd    20000000000000000000000
  MIN_HEALTH_FACTOR                   1000000000000000000
  USER's HealtheFactor                115792089237316195423570985008687907853269984665640564039457584007913129639935
  
  Post - MintDSC  
  ----------------------------------------------------
  USER DSC balance                    100000000000000000000
  USER weth balance                   0
  USER's TotalDscMinted               100000000000000000000
  USER's TotalCollateralValueInUsd    20000000000000000000000
  MIN_HEALTH_FACTOR                   1000000000000000000
  USER's HealtheFactor                100000000000000000000
